README.md

```diff
@@ -1 +1,48 @@
-# community
\ No newline at end of file
+# ROS 2 Security Working Group
+The Security Working Group's mission is to advocate for and implement security features within ROS 2. The working group negotiates a balance between the benefits of security and the enablement of technology, striving for a secure-by-default design which can be customized to suit a targeted security profile.
+
+This document outlines the governance of the ROS 2 Security Working Group. Updates to this document will be handled in the same manner as other project updates governed by the Working Group.
+
+## Subprojects
+A significant portion of the Security Working Group's mission is achieved by maintaining ROS projects. The following projects are owned by this Working Group:
+* _TBD:  list of project names_
+
+### Adding subprojects
+To request that the Security Working Group take on ownership and maintainership of a particular project, create a new issue in this repository using the appropriate issue template. You may be requested to present your proposal at the next Security Working Group meeting. Approvers will vote on whether or not the Working Group will accept the project.
+
+### Standards for subprojects
+Subprojects must meet the following criteria:
+* Builds must pass against ROS 2 master
+* Test suite must pass
+* Test coverage must be greater than 50%
+* The ROS 2 standard linter set must be enabled and adhered to
+* Builds must have 0 warnings
+
+## Governance
+These guidelines are designed exclusively to help the Working Group achieve its mission, but are subject to change should they become cumbersome or ineffective.
+
+### Meetings
+* The working group typically meets on the last Tuesday of each month, at alternating times to accomodate our community's varied timezones
+* Meetings are announced and an agenda created on the ROS Discourse using the [wg-security tag](https://discourse.ros.org/tags/wg-security)
+* Meeting notes are kept on the [ROS Wiki](http://wiki.ros.org/ROS2/WorkingGroups/Security)
+* Meetings are recorded and available [on YouTube](https://www.youtube.com/playlist?list=PLpUh4ScdBhSMaEekJ8xeAAGmWUgR9S1K_)
+
+### Communication channel
+The Security Working Group has a few communication channels:
+* ROS Discourse using the [wg-security tag](https://discourse.ros.org/tags/wg-security)
+* Chat using a [Matrix room](https://matrix.to/#/!LcRLnAIRWjSCfZmMeD:matrix.org?via=matrix.org)
+
+### Roles
+Security Working Group members may act in one or more of the following roles:
+* __Member__
+  * Attend a majority of the Security Working Group meetings
```

Comment:
If meeting alternate between timezones, it may be challenging for members of the timezones covered only every other meeting to meet the "majority of meetings" criterium

Reply:
+1 for this. As we rotate the time, it should be reasonable that only timelines within a somewhat professional time-range are accounted for.

AFAIK, this is somehow the policy that Open Robotics is using for the ROS 2 TSC.

Reply:
That's fair, we could adjust the rotation a bit. I'm nervous about playing too much with the requirement though: I think it's important for members to be limited to folks who are coming to meetings consistently, otherwise it's difficult to work as a team. Remember that non-members can still participate; membership comes with responsibility, and I think it's best if we avoid relying on folks we can never talk to.

Reply:
> The TSC says you can miss up to three meetings in a row, I think. We could move to something like that.

That seems fair. On a round robin rotation you would expect that 2 out of 3 meeting can be attended by members regardless of their timezone.

Reply:
Alright, updated.